lib/shadow.mli

```diff
@@ -16,6 +16,13 @@ val to_string : t -> string
 
 type db = t list
 
+type shadow_t
+
+val shadow_t : shadow_t Ctypes_static.structure Ctypes.typ
```

Comment:
I think this is unused, but its obsoleted anyway by the followup commit that introduces Mem